docs/platforms/javascript/common/feature-flags/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Set Up Feature Flags
+sidebar_order: 7000
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+description: "Learn how to set up feature flag evaluation tracking and feature flag change tracking."
+---
+
+<Alert level="info" title="Currently in Beta">
+
+The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in beta.
+
+</Alert>
+
+Link your external feature flag integrations with Sentry to provide feature flag insights right inside the Sentry UI. Linking one or more integrations will allow you to view recent flag evaluations in one place and identify potential suspect flags related to errors. In addition, Sentry will provide insights on feature flag updates relative to error event timelines.
+
+To set up **feature flag evaluation tracking**, you will need to set up your language-specific SDK to include Sentry's feature flag integration. 
+
+Learn more about the integrations available for JavaScript and how to set them up:
+- [OpenFeature](/platforms/javascript/integrations/openfeature/)
+- [LaunchDarkly](/platforms/javascript/integrations/launchdarkly/)
+
+Please read the note below to ensure that you also complete one additional step.
+
+<Alert level="warning" title="Note">
+
+In order to take full advantage of the feature flag capabilities Sentry offers, there is an additional setup step needed, which is setting up your integration-specific webhook. This is needed to enable **feature flag change tracking**, so that your integration may communicate feature flag changes to Sentry. Learn how to set this up by [reading the docs](/product/explore/feature-flags/#set-up-your-integration-specific-webhook). 
```

Comment:
```suggestion
In order to take full advantage of the feature flag capabilities Sentry offers, there is an additional set up step needed, which is setting up your integration-specific webhook. This is needed to enable **feature flag change tracking**, so that your integration can communicate feature flag changes to Sentry. Learn how to set this up by [reading the docs](/product/explore/feature-flags/#set-up-your-integration-specific-webhook). 
```